orderer/common/localconfig/config.go

```diff
@@ -212,6 +214,12 @@ type ChannelParticipation struct {
 	MaxRequestBodySize uint32
 }
 
+// Message size for the GRPC server and client.
+type Message struct {
```

Comment:
We already have connection configuration elements at the top level -- Keepalive, ConnectionTimeout -- any reason to push this into its own structure instead of hanging them off the top level?  As written, the name stutters as "Message.MaxRecvMsgSize".

Reply:
That makes more sense to me. Thanks!